pkg/volume/util/volumepathhandler/volume_path_handler.go

```diff
@@ -225,7 +226,7 @@ func (v VolumePathHandler) RemoveMapPath(mapPath string) error {
 		return fmt.Errorf("failed to remove map path. mapPath is empty")
 	}
 	klog.V(5).Infof("RemoveMapPath: mapPath %s", mapPath)
-	err := os.RemoveAll(mapPath)
+	err := syscall.Rmdir(mapPath)
```

Comment:
Why not just use ```os.Remove```

Reply:
os.Remove calls syscall.Rmdir. And since we know this is a directory, why not? And we also have some rmdir calls in kubelet now.